src/pages/TasksPage.js

```diff
@@ -26,9 +26,12 @@ function TasksPage() {
 
   const handleAdd = (event) => {
     event.preventDefault();
-
-    setSavedValue([...savedValue, value]);
-    setValue("");
+    if (value) {
+      setSavedValue([...savedValue, value]);
+      setValue("");
+    } else {
+      alert("Please enter To Do..");
```

Comment:
We need to use `a` before `To Do`.
The message should be "Please enter a To Do...".
Here is an explanation: https://chat.openai.com/share/0282ace9-252b-4af5-84f4-c0ed7f802f4f